blocks/init/src/Blocks/wrapper/components/wrapper-toolbar.js

```diff
@@ -0,0 +1,60 @@
+    };
+
+    const copyAttributes = () => {
+        localStorage.removeItem('copiedWrapperAttributes');
+
+        const copiedWrapperAttributes = Object.keys(attributes).filter((key) => key.includes('wrapper'))
+        .reduce((cur, key) => { 
+            return Object.assign(cur, { [key]: attributes[key] });
+        }, {});
+
+        localStorage.setItem('copiedWrapperAttributes', JSON.stringify(copiedWrapperAttributes));
+    };
+
+    const pasteAttributes = () => {
+        const wrapperAttributesToBePasted = JSON.parse(localStorage.getItem('copiedWrapperAttributes'));
+        setAttributes(wrapperAttributesToBePasted);
+    };
+
+	return (
+		<>
+            <DropdownMenu 
+                icon={iconCopy()}
+                label={__('Copy/Paste wrapper attributes', 'eightshift')}
+                className="components-toolbar es-toolbar-icon-24"
+            >
+                { ( { onClose } ) => (
+                    <Fragment>
+                        <MenuGroup>
+                            <MenuItem onClick={ () => {copyAttributes(); onClose();} } icon={iconCopy()}>
+                                <button className="components-button components-dropdown-menu__toggle has-icon">
+                                    {__('Copy Wrapper Attributes', 'eightshift')}
+                                </button>
+                            </MenuItem>
+                        </MenuGroup>
+                        <MenuGroup>
+                            <MenuItem onClick={ () => {pasteAttributes(); onClose();} } icon={iconPaste()}>
+                                <button className="components-button components-dropdown-menu__toggle has-icon">
+                                    {__('Paste Wrapper Attributes', 'eightshift')}
+                                </button>
+                            </MenuItem>
+                        </MenuGroup>
+                    </Fragment>
+                ) }
+            </DropdownMenu>
+        </>
```

Comment:
This structure seems a bit odd and overly complex, please check the Gutenberg documentation.
The text domain should be `eighsfhit-frontend-libs` if I remember correctly.
Also not a fan of the inline click handlers.

We should definitely consider putting this in the Options panel, I have a feeling the toolbar is starting to get a bit too cluttered, and this will just add to the confusion.

Reply:
Maybe add this to the `...` dropdown menu in the toolbar?

Reply:
Looking at the Block Editor Handbook example, the markup they give as one of the examples is much more logical:
```jsx
import { Fragment } from '@wordpress/element';
import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
import { more, arrowUp, arrowDown, trash } from '@wordpress/icons';
 
const MyDropdownMenu = () => (
    <DropdownMenu icon={ more } label="Select a direction">
        { ( { onClose } ) => (
            <Fragment>
                <MenuGroup>
                    <MenuItem icon={ arrowUp } onClick={ onClose }>
                        Move Up
                    </MenuItem>
                    <MenuItem icon={ arrowDown } onClick={ onClose }>
                        Move Down
                    </MenuItem>
                </MenuGroup>
                <MenuGroup>
                    <MenuItem icon={ trash } onClick={ onClose }>
                        Remove
                    </MenuItem>
                </MenuGroup>
            </Fragment>
        ) }
    </DropdownMenu>
);
```

It could be even a bit simpler in this case:
```jsx
import { __ } from '@wordpress/i18n';
import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
import { icons } from '@eightshift/frontend-libs/scripts';

const ComponentDemo = () => {
	//

	const copyWrapperConfig = (closeMenu) => {
		// ...

		closeMenu();
	};
	
	const pasteWrapperConfig = (closeMenu) => {
		// ...

		closeMenu();
	};

	return (
		<DropdownMenu icon={icons.wrapperConfig} label={__('', 'eightshift-frontend-libs')}>
			{({ onClose }) => (
				<MenuGroup>
					<MenuItem icon={icons.copy} onClick={() => copyWrapperConfig(onClose)}>
						{__('Copy configuration', 'eightshift-frontend-libs')}
					</MenuItem>
					<MenuItem icon={icons.paste} onClick={() => pasteWrapperConfig(onClose)}>
						{__('Paste Configuration', 'eightshift-frontend-libs')}
					</MenuItem>
				</MenuGroup>
			)}
		</DropdownMenu>
	);
};
```

Reply:
I've refactored this, thanks for clarifying.